cachi2/core/package_managers/rpm/main.py

```diff
@@ -216,7 +218,8 @@ def fetch_rpm_source(request: Request) -> RequestOutput:
         # We're deliberately taking the easy route here by only assuming the "last" set of options
         # we found in the input JSON instead of doing a deep merge of all the nested dicts.
         # Nevertheless, we'll at least emit a warning at the end so that the user is informed
-        if package.options and package.options.dnf:
+
+        if package.options and package.options.dnf:  # type: ignore
```

Comment:
Does mypy still complain? I'm not seeing any errors, so the type ignore should likely be dropped.

Reply:
removed it